src/cmd/cloud.js

```diff
@@ -341,20 +365,28 @@ module.exports = class CloudCommand extends CLICommandBase {
 					for (let i = 0, n = list.length; i < n; i++){
 						this.ui.stdout.write(`    ${list[i]}${os.EOL}`);
 					}
+					if (assets) {
+						this.ui.stdout.write(`Including assets:${os.EOL}`);
+						_.values(assets).forEach((asset) => {
+							this.ui.stdout.write(`    ${asset.name}${os.EOL}`);
```

Comment:
Let's print the path to assets as well
```suggestion
							this.ui.stdout.write(`    ${asset.path}${os.EOL}`);
```

Then you can remove the `Including assets:` line above and it will print a combined list of source code and assets. It should be clear which is which because the assets are last and start with `assets/`

Note that you'll need to update `getAssets` to set the `path` property to `path.join(assetsPath, name)`